lib/presentation/models/word_cloud/word_cloud_data.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/painting.dart';
+import 'dart:ui';
+
+class FlutterHashtag {
+  const FlutterHashtag(
+    this.hashtag,
+    this.color,
+    this.size,
+    this.rotated,
+  );
```

Comment:
Add a newline below

Reply:
Addressed in commit c5eb15f9e6ba6e2af44ba035a37cdde27d4c5633